src/Admin/Routing/Extension/RouteReferrersExtension.php

```diff
@@ -26,7 +26,7 @@ class RouteReferrersExtension extends AbstractAdminExtension
     private $formGroup;
     private $formTab;
 
-    public function __construct($formGroup = 'form.group_routes', $formTab = 'form.tab_routing')
+    public function __construct($formGroup = 'form.group_routing', $formTab = 'form.tab_routing')
```

Comment:
why not group_routes? its the list of routes, not the routing system...

as we have defaults in the configuration class as well, i would make the arguments required here and have defaults in only one place.